app/controllers/api/v1/users_controller.rb

```diff
@@ -14,9 +14,9 @@ def show
   def index
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.Style/SpaceInsideHashLiteralBraces
**Severity**: INFO
**File**: app/controllers/api/v1/users_controller.rb L14

Space inside } missing.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*